anatomy/tracks/concept-exercises.md

```diff
@@ -192,12 +206,6 @@ Viewing hints will not be a "recommended" path and we will (softly) discourage u
 [methods]: https://launchschool.com/books/ruby/read/methods
 ```
 
-### File: .docs/source.md (required if there are third-party sources)
-
-**Purpose:** Describe the third-party source(s) of the exercise.
-
-This file contains third-party references and sources of the exercise. Only required if there are any such sources, but not if the exercise was completely designed from scratch for Exercism.
-
```

Comment:
This file will be replaced by two new fields in the `.meta/config.json` file. There were only a couple of usages of this file, so migrating should not be an issue.